package/MDAnalysis/core/universe.py

```diff
@@ -992,6 +993,194 @@ def add_Segment(self, **attrs):
         self.segments = SegmentGroup(np.arange(self._topology.n_segments), self)
         # return the new segment
         return self.segments[segidx]
+    
+    def _add_TopologyObjects(self, object_type, values, types=None, guessed=False,
```

Comment:
Back when it was called `add_TopologyObject` the naming followed the `Universe.add_Residue`, `Universe.add_Segment`, `Universe.add_TopologyAttr`. Lower-casing it now because the method adds values to `Bonds` etc instead of adding a `Bonds` class (usually)